README.md

```diff
@@ -213,6 +213,7 @@ The supported rules are described below:
 | no_array_responses           | Flag any operations with a top-level array response.                                | shared   |
 | parameter_order              | Flag any operations with optional parameters before a required param.               | shared   |
 | operation_id_naming_convention | Flag any `operationId` that does not follow naming convention.                    | shared   |
+| undefined_required_properties| Flag any operations with undefined required properties                              | shared   |
```

Comment:
We aren't quite flagging "operations" so I think this description could be more accurate - we're flagging properties of a schema.

This comment is making me realize this configuration option belongs under the "schemas" category rather than the "operations" category - my apologies for not catching this earlier